### PR TITLE
Refactor to use EDC s3 bucket

### PR DESCRIPTION
**Pull request by jacquelynsmale**

### CODE REVIEW

File: image_services/egis/make_egis_services.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def main():
     parser.add_argument('config_file')
     args = parser.parse_args()
 
-    bucket = 'sentinel-1-global-coherence-earthbigdata'
+    bucket = 'asf-ngap2w-p-s1-global-coherence'
     overview_path = '/vsis3/asf-gis-services/public/GSSICB/'
```
Comment (jacquelynsmale) [inline image_services/egis/make_egis_services.py:178]: Are we still going to store overviews in our s3 bucket?

Comment (asjohnston-asf) [reply image_services/egis/make_egis_services.py:178]: Yes, I think that's the best approach for the short term. In a perfect world all data related to the image service (both the geotiffs and the overviews) would be stored in Earthdata Cloud, but I don't see a pressing need to move where the overviews are stored right now.

That does mean we'll have to use one set of AWS keys when reading geotiff data from the EDC bucket (building the csv, creating the mosaic, creating the overview) and a separate set of AWS keys when uploading the overview. That probably means some updates to the python script and maybe some new command line parameters.

File: image_services/egis/config/COH06_VV_DJF.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "project_name": "GSSICB",
-  "s3_prefix": "data/tiles",
+  "s3_prefix": "tiles/N00",
```
Comment (asjohnston-asf) [inline image_services/egis/config/COH06_VV_DJF.json:3]:
```suggestion
  "s3_prefix": "tiles",
```

File: image_services/egis/make_egis_services.py [hunk 3]
```diff
@@ -192,16 +197,17 @@ def main():
 
     arcpy.env.parallelProcessingFactor = '75%'
 
-    rasters = get_rasters(bucket, config['s3_prefix'], config['s3_suffix'])
-    update_csv(csv_file, rasters)
+    try:
```
Comment (asjohnston-asf) [inline image_services/egis/make_egis_services.py:200]: I think we can move this `try` back to it's original location, right before creating the geodatabase?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,14 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.6.2]
+## [0.6.3]
```
Comment (asjohnston-asf) [inline CHANGELOG.md:7]: production is at 0.6.1, so we can lump all these entries under 0.6.2

Comment (jacquelynsmale) [reply CHANGELOG.md:7]: I think we recently released [0.6.2](https://github.com/ASFHyP3/gis-services/blob/main/CHANGELOG.md#062), so it should be 0.6.3

File: image_services/egis/make_egis_services.py [hunk 5]
```diff
@@ -171,10 +175,11 @@ def main():
 
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('--working-directory', default=os.getcwd())
+    parser.add_argument('--edl-token', default='/home/arcgis/edl_token.json')
```
Comment (asjohnston-asf) [inline image_services/egis/make_egis_services.py:178]:
```suggestion
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**